src/mac.rs

```diff
@@ -514,64 +525,165 @@ impl Security {
     pub fn from_bits(bits: u8) -> Option<Self> {
         match bits {
             0b0 => Some(Security::None),
-            _   => None,
+            _ => None,
         }
     }
 }
 
-
-/// Defines whether PAN ID compression is enabled
-///
-/// Part of [`Header`]. PAN ID compression is currently not supported by this
-/// implementation.
+/// Defines version information for a frame
 #[derive(Clone, Copy, Debug, Eq, PartialEq)]
-pub enum PanIdCompress {
-    /// PAN ID compression is disabled
-    Disabled = 0b0,
+pub enum FrameVersion {
+    /// A frame conforming to the 802.15.4-2003 standard
+    Ieee802154_2003 = 0b00,
+    /// A frame conforming to the 802.15.4-2006 standard
+    Ieee802154_2006 = 0b01,
+    /// A frame conforming to the current 802.15.4 standard
+    Ieee802154 = 0b10,
```

Comment:
Shouldn't that be `Ieee802154_2015`, or whatever the latest version is?

Reply:
The IEEE 802.15.4-2015 standard document seems to describe 0b10 as the current version described in the standard document.